docs/docsource/operation/XDM_GUI/location_management/add_a_storage_location.rst

```diff
@@ -0,0 +1,272 @@
+.. _Add a Storage Location:
+
+Add a Storage Location
+======================
+
+To add a storage location:
+
+1. Click the **Storage Locations** item in the sidebar to open the
+   **Cloud Locations** window:
+
+   .. image:: ../../Graphics/Orbit_Storage_Locations.png
+      :align: center
+
+#. Click **Add New**.
+
+#. The **Add New Storage Location** dialog displays:
+
+   .. image:: ../../Graphics/Orbit_Add_New_Storage_Location.png
+      :align: center
+
+   a. Enter a location name in the **Location Name** field using
+      lowercase letters, numbers, and dashes.
+
+      .. note::
+
+         Capital letters, spaces, and punctuation and diacritical
+         marks will result in an error message.
+
+   b. Select a location type from the **Location Type** pull-down menu.
+      You can choose:
+
+      * Amazon S3
+      * DigitalOcean Spaces
+      * Wasabi
+      * Google Cloud Storage
+      * Microsoft Azure Blob Storage
+      * NFS Mount
+      * Scality RING with S3 Connector
+      * Scality RING with sproxyd Connector
+      * Ceph RADOS Gateway
+      * A |product| local filesystem
+
+#. Each storage location type has its own requirements. No security is
+   required for a local file system, but all public clouds require
+   authentication information.
+
+   .. note::
+
+      Adding a location requires credentials (an access key and a secret key).
+      Though nothing prevents you from using account-level credentials when
+      |product| requests credentials for a location, it is a best practice to enter
+      credentials specifically generated for this access. In other words, before
+      you add a location, first create a user in that location (an AWS account
+      or an S3 Connector, for example) for the purpose of |product| access. Give
+      that |product| "user" all and only the permissions needed to perform the
+      desired tasks.
+
+   .. tip::
+   
+      When configuring an S3 Connector, assign the following policy to the
+      special zenko-access user to ensure access to the Metadata service and the
+      ability to perform operations on the bucket:
+
+   ::
```

Comment:
missed the json tag change here

Reply:
It has not been my practice to use these, so it's not so much a "ya missed" as an "add this". ;-)

Reply:
oh I was just referencing my comment from the other PR where you did add them. But fair point.

Reply:
I guess we saw this in 3057 and it dropped out when I rebased. Adding it in again. Probably will get a conflict on the spacing...